app/templates/src/main/java/package/security/_AppUser.java

```diff
@@ -0,0 +1,99 @@
+package <%=packageName%>.security;
+
+import java.util.Collection;
+import java.util.Set;
+
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+/**
+ * Class to represent the authenticated user to the spring security.
+ * 
+ * @see UserDetails
+ * @author jhipster
+ *
+ */
+public class <%= capitalizedBaseName %>User implements UserDetails {
+
+    private static final long serialVersionUID = 1L;
+
+    private final <%= pkType %> id;
```

Comment:
We don't use the final keyword in JavaBeans (can be discussed why, but here it's just to be consistent with the rest of the code)